### ISSUE TITLE
Selection of NAT or Host-only for Master Honeyclient VM

### ISSUE THREAD

**Issue by dkindlund** [dkindlund/honeyclient#177]
I follow the instruction to install HoneyClient environment and has some question:
According to network architecture diagram, the HoneyClient is located at 
vmnet1 - Host Only class-C network (10.0.0.1/24)

but the installation statement of "Create Master Honeyclient VM"
instruct to use
"Use network address translation (NAT) as a network connection."

which network is correct NAT (vmnet8) or Host-only (vmnet1) should apply?

**Comment by dkindlund** [comment]
Author: kindlund
I assume you're talking about Step 1 in this section:

http://www.honeyclient.org/trac/wiki/UserGuide#HoneyclientVM

The issue is that you need to "prepare" the master VM by downloading and installing software inside this VM.  In order to accomplish this, we assume you want the master VM to 'temporarily' have network connectivity -- just to accomplish this.

So, we recommend that you TEMPORARILY switch the master VM's network settings over to NAT (vmnet8), download/install all the software listed (e.g., Cygwin packages), and then switch the master VM's network settings BACK to Host-only (vmnet1).

The section where we indicate that you switch the master VM back to Host-only (vmnet1) is listed here:

http://www.honeyclient.org/trac/wiki/UserGuide#FinalizeMasterVMSettings

In short: Normally, yes, the master VM's network setting is set to Host-only (vmnet1).  However, we realize that you may want to temporarily use the network to "prepare" the master VM.

Does this make sense?

-- Darien

**Comment by dkindlund** [comment]
Author: kindlund

**Comment by dkindlund** [comment]
Author: kindlund
Closing ticket, since there are no new issues to report.
